CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ This project **does not** adhere to [Semantic Versioning](http://semver.org/).
 This file tries to follow the conventions proposed by [keepachangelog.com](http://keepachangelog.com/).
 Here, the categories "Changed" for added and changed functionality,
 "Fixed" for fixed functionality, and
-"Removed" for removed functionality is used.
+"Removed" for removed functionality are used.
```

Comment:
Why "are"?
Functionality is singular 
Either you let the is or you use functionalities